azext_iot/_help.py

```diff
@@ -1247,6 +1247,25 @@
         az iot central app validate-messages --app-id {app_id} --simulate-errors
   """
 
+helps[
+    "iot central app show-provisioning-info"
+] = """
+    type: command
+    short-summary: get provisioning information of the devices associated with the app.
+    long-summary: |
+                  EXPERIMENTAL requires Python 3.5+
```

Comment:
You don't need requires Python 3.5 statement anymore. It's assumed since we dropped Py3.4 and Py2.7 support. You are also not using uamqp for this command right?